CMakeLists.txt

```diff
@@ -57,6 +57,24 @@ if(NOT BUILD_SYSTEM_DEPS IN_LIST BUILD_SYSTEM_DEPS_LIST)
     message(FATAL_ERROR "BUILD_SYSTEM_DEPS must be one of ${BUILD_SYSTEM_DEPS_LIST}")
 endif()
 
+if ((BUILD_READERS OR BUILD_WRITERS) AND (BUILD_EXAMPLES OR BUILD_BLACKBOX_TESTS))
+
+    if (BUILD_SYSTEM_DEPS STREQUAL "AUTO")
+        find_package(PkgConfig)
+        pkg_check_modules (STB IMPORTED_TARGET stb)
+    elseif (BUILD_SYSTEM_DEPS STREQUAL "ALWAYS")
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules (STB REQUIRED IMPORTED_TARGET stb)
+    endif()
+
+    if (NOT STB_FOUND)
+        include (${CMAKE_SOURCE_DIR}/FetchSTB.cmake)
+    else()
+        add_library(stb::stb ALIAS PkgConfig::STB)
```

Comment:
nice: I was about to suggest that change before already, but did not want to be too pedantic ;)

Reply:
I had neglected to test BUILD_SYSTEM_DEPS=AUTO with stb not installed, luckily the CI tests that scenario :)